commands/createfixpullrequests.go

```diff
@@ -351,15 +364,31 @@ func (cfp *CreateFixPullRequestsCmd) updatePackageToFixedVersion(impactedPackage
 			}
 		}()
 	}
+	if buildToolDependency := cfp.isBuildToolsDependency(fixDetails); buildToolDependency {
+		log.Info("Cannot update build tools dependencies, skipping...")
```

Comment:
log.Info --> log.Debug
Cannot --> Should not...
Let's also add the package name and version to this debug message.